openkfw/TruBudget
# Provisioning: Add Groups

# Description
To test the functionality of groups on API level, groups are added to the provisioning of the blockchain. For this purpose, 2 groups are added: 
* Admins - Created with 2 users already in the API call
* Reviewers - Created without users, where 3 users are added and one is removed

Closes #57 

# How to test
Start an empty deployment and then run the provisioning. The groups should then be visible on the groups page of TruBudget (or listed when the API endpoint `/group.list` is called)
